### PR TITLE
Updating to Astro 5.1

### PR DESCRIPTION
**Pull request by dmcalester**
- New guidance on Sign In patterns
- Updated guidance on Form Fields and validation aligning with Kobayashi Maru 1.2 guidance
- Fixed incorrect font size guidance

### CODE REVIEW

**Comment by dmcalester** [comment]
Preparing to publish 5.1

**Comment by amalyah** [comment]
The "Propose a change or fix to this page" link appears to be broken throughout the site.

**Comment by dmcalester** [comment]
> The "Propose a change or fix to this page" link appears to be broken throughout the site.

That link is dynamically generated and linked to the git branch. Unfortunately as deploy previews are ephemeral and don’t have a branch those links won’t work.

**Comment by amalyah** [comment]
Design Process images are still blurry. (Known issue I think, but thought I'd mention it in case it wasn't.)

**Comment by amalyah** [comment]
Note for all GRM/TT&C pages: #contentBottom links no longer works and Bitbucket link at the bottom is no longer accurate.

**Comment by amalyah** [comment]
Image issue: _content/img/patterns/val-required-input.png

Second input (the invalid one) needs to have the text content changed from a value back to placeholder text so that the context of the field being invalid because it's a required field makes sense.

**Review by amalyah — CHANGES_REQUESTED** [review]
See comments.

File: _content/components/switch.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+tags: components
+path: /components/switch
+date: Last Modified
+layout: components.template.njk
+title: Switch
+demo: components-switch--switch
+storybook: components-switch--switch
```
[the remark this answers is not in the document]
Comment (dmcalester) [reply _content/components/switch.md:8]: Known issue, it shouldn’t be an issue when it leaves preview

File: netlify.toml [hunk 2]
```diff
@@ -39,6 +39,12 @@
 	to = "/patterns/forms-and-validation"
 	status = 301
 	force = true
+	
+[[redirects]]
```
[the remark this answers is not in the document]
Comment (dmcalester) [reply netlify.toml:43]: Weird it’s in `master`

File: _content/components/sign-in.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+# Sign In
+
+The Sign in page is often the first interaction a user has with your product. This entry point experience is an important moment in establishing your product’s brand and experience and sets the tone for their overall experience with the product.
+“Signing in” provides an authentication method to validate a user’s unique identification and allow them access to protected areas of an application. A user provides their required identification and, if correct, they are provided access to the product.
+
+ ::: caution
+ When implementing validation and security measures Astro users should consult with their organization's IT/Security departments and adapt these patterns to suit the requirements of their organization’s best practices and security needs.
+ :::
+
+![Example of a full-page, simple sign in screen](/img/components/signin-simple-full.png "Example of a full-page, simple sign in screen")
```
[the remark this answers is not in the document]
Comment (dmcalester) [reply _content/components/sign-in.md:23]: Deferred

File: _content/design-guidelines/theme.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ Individual sections or components can override their inherited theme on a case-b
 
 ## Future Considerations
 
-While not enabled by default at this time, light and dark themes can be swapped at runtime. For situations where lighting conditions can vary considerably, consider enabling runtime theme switching. A user-definable toggle in the application settings, an ambient light sensor or respecting OS level dark/light theme settings are all viable options.
+While not enabled by default at this time, light and dark themes can be swapped at runtime. For situations where lighting conditions can vary considerably, consider enabling runtime theme switching. A user-definable switch in the application settings, an ambient light sensor, or respecting OS level dark/light theme settings are all viable options.
 
 ## Implementation
```
[the remark this answers is not in the document]
Comment (dmcalester) [reply _content/design-guidelines/theme.md:50]: Deferred

File: _content/components/__template.md [hunk 5]
```diff
@@ -1,12 +1,14 @@
...
-path: /components/component-template
+path: /components/template
 date: Last Modified
 layout: components.template.njk
-title: Component Template
-demo: https://rocketcom.bitbucket.io/html-demos/component-template.html
-storybook: components-story--component-template
-git: component-template/
+title: Template
+demo: components-buttons--all-button-variants
```
[the remark this answers is not in the document]
Comment (dmcalester) [reply _content/components/__template.md:7]: It’s an area for us to test. It doesn’t get published.

File: _content/getting-started/designers.md [hunk 6]
```diff
@@ -27,7 +27,7 @@ To download the Design starter kit, navigate to Astro Design Resources [Git Repo
 Download files from Astro Sketch Libraries:
 
 [Astro Dark Library](https://github.com/RocketCommunicationsInc/astro-design-resources/raw/master/Sketch/Astro%20Dark%20Library.sketch)
-[Astro Light Library](https://github.com/RocketCommunicationsInc/astro-design-resources/raw/master/Sketch/Astro%20Light%20Library.sketch)
+[Astro Light Library](https://github.com/RocketCommunicationsInc/astro-design-resources/raw/master/Sketch/Astro%20Dark%20Library.sketch)
```
[the remark this answers is not in the document]
Comment (amalyah) [reply _content/getting-started/designers.md:30]: @dmcalester This still appears to be an issue. Should we address it or defer it?

File: _content/getting-started/_readme.md [hunk 7]
```diff
@@ -53,16 +53,19 @@ Last updated: {{ page.date }}
 
 Astro is provided as an open-source tool under the **Astro Space UX Design System Interim License Agreement**:
 
+::: caution When implementing Astro Space UX Design System users should consult with their organization's IT/Security departments and adapt these patterns to suit the needs of their organization’s best practices and security requirements.
```
[the remark this answers is not in the document]
Comment (amalyah) [reply _content/getting-started/_readme.md:56]: @dmcalester This is still an issue. Should we address it or defer?

File: _content/components/checkbox.md [hunk 8]
```diff
@@ -13,21 +13,24 @@ theme: true
...
 ## Appearance and Behavior
 
-A checkbox can be configured for required input. [See Form and Input Validation](/patterns/forms-and-validation).
+A basic Checkbox consists of a visual indicator of its selected state followed by a label. Individual Checkboxes can appear selected (checked), Unselected (no check), and Disabled (no action can be taken by the user). An Indeterminate state (a dash symbol rather than checked) may display when a checkbox is used as a parent of a group of checkboxes where at least one child is selected and at least one is not.
+
+A checkbox can be configured for required input. Help Text for individual Checkbox list items left-aligns with the item’s text and not its icon for easier text scanning. To learn more about adding Help Text or Validation to Checkboxes or Checkbox groups, see the [Forms and Validation](/patterns/forms-and-validation) guidance.
 
 ::: note
-Don’t use a Checkbox to initiate an action. Instead, use an [Action Button](/components/button) or a [Toggle Button](/components/toggle)
+Don’t use a Checkbox to initiate an action. Instead, use an [Action Button](/components/button) or a [Switch Button](/components/switch)
```
Comment (amalyah) [inline _content/components/checkbox.md:33]: Is the component technically a Switch or a Switch Button? (I thought it was Switch, but I may be incorrect).

File: _content/fd-service-ux-design/about-the-fd-designs.md [hunk 9]
```diff
@@ -104,14 +104,14 @@ For operators of the FDS application, flexibility and automating redundant tasks
 
 ### Global Status Bar
 
-The [Global Status Bar](../../components/global-status-bar) of the FDS app contains a single login, high level navigation [tabs](../../components/tabs) to switch between tools, a [clock](../../components/tabs) and notification, chat and help icons. [Monitoring icons](../../components/icons-and-symbols) may be included if the mission requires additional situational awareness.
+The [Global Status Bar](../../components/global-status-bar) of the FDS app contains a single sign in, high level navigation [tabs](../../components/tabs) to switch between tools, a [clock](../../components/tabs) and notification, chat and help icons. [Monitoring icons](../../components/icons-and-symbols) may be included if the mission requires additional situational awareness.
```
[the remark this answers is not in the document]
Comment (amalyah) [reply _content/fd-service-ux-design/about-the-fd-designs.md:107]: @dmcalester This is still an issue. Should this be addressed or deferred?

**Review by amalyah — CHANGES_REQUESTED** [review]
Still pending some resolution or decision on deferment on issues found in last review related to the following pages:
_content/getting-started/designers.md
_content/getting-started/_readme.md
_content/fd-service-ux-design/about-the-fd-designs.md
_content/components/checkbox.md

File: _content/getting-started/_readme.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (amalyah) [reply _content/getting-started/_readme.md:56]: @dmcalester Agreed that this should be addressed before merge since it looks actively broken.

**Review by amalyah — CHANGES_REQUESTED** [review]
Approved pending fix to caution note in the Getting Started Read Me page preview.

**Comment by dmcalester** [comment]
@Cheshire89 @amalyah @cskerbo @Full-lifey Amalya has done a review. Just looking for anything show-stopping at this point.

**Review by amalyah — APPROVED** [review]

**Pull request merged**